doc/changelog/02-specification-language/18254-master+def-in-context-instance-in-modtype.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:**
+  Definitions done using :cmd:`Context` outside a section in a
```

Comment:
Maybe we could emphasize that this is about definitions (ie with `:= body`) vs assumptions, I got confused for a bit.

Reply:
You mean something like that:
```suggestion
  Declarations of the form `id := body` in :cmd:`Context` outside a section in a
```

Reply:
yes